packages/odyssey-babel-loader/src/index.js

```diff
@@ -0,0 +1,43 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+module.exports = require("babel-loader").custom(() => ({
+  customOptions({ include, loader = {} }, { source }) {
+    const dependencyRegexp = buildImportDependencyRegexp(include);
+
+    Array.from(source.matchAll(dependencyRegexp)).forEach(([, dependency]) => {
+      const absoluteDependency = this.utils.absolutify(
+        this.context,
+        dependency
+      );
+      this.addDependency(absoluteDependency);
+    });
+
+    return { loader };
+  },
+}));
+
+function buildImportDependencyRegexp(include) {
+  const normalized = Array.isArray(include)
+    ? include.map(toRegexpString)
+    : ["\\.module\\.(?:scss|css)"];
```

Comment:
🤣 I read this wrong with the `?` in my earlier comment. The change to make it explicit makes it very clear.

👍